test/viewModels/drafts/AllDraftDeparturesViewModelSpec.scala

```diff
@@ -287,13 +287,23 @@ class AllDraftDeparturesViewModelSpec extends SpecBase with Generators with Scal
       val departuresSummary   = arbitrary[DeparturesSummary].sample.value
       val paginationViewModel = DraftsPaginationViewModel(2, 1, 2, "test")
 
+      val viewModel = AllDraftDeparturesViewModel(departuresSummary, 1, None, frontendAppConfig.draftDepartureFrontendUrl, paginationViewModel)
+
+      val sortMessageLRNDescending = messages("departure.drafts.dashboard.table.sort.hidden", viewModel.referenceNumber, "descending")
+      val sortMessageLRNDefault    = messages("departure.drafts.dashboard.table.sort.hidden", viewModel.referenceNumber, "ascending")
+
+      val sortMessageDaysToCompleteAscending = messages("departure.drafts.dashboard.table.sort.hidden", viewModel.daysToComplete, "ascending")
+      val sortMessageDaysToCompleteDefault   = messages("departure.drafts.dashboard.table.sort.hidden", viewModel.daysToComplete, "descending")
+
       "when sortParams is SortByLRNAsc" in {
         val sortParams = SortByLRNAsc
         val viewModel  = AllDraftDeparturesViewModel(departuresSummary, 1, None, frontendAppConfig.draftDepartureFrontendUrl, paginationViewModel, sortParams)
         viewModel.sortLrn mustBe "ascending"
         viewModel.sortCreatedAt mustBe "none"
         viewModel.sortLRNHref() mustBe routes.DashboardController.onPageLoad(None, None, Some(SortByLRNDesc.toString))
         viewModel.sortCreatedAtHref() mustBe routes.DashboardController.onPageLoad(None, None, Some(SortByCreatedAtDesc.toString))
+        viewModel.sortHiddenTextLRN(messages) mustBe sortMessageLRNDescending
+        viewModel.sortHiddenTextDaysToComplete(messages) mustBe sortMessageDaysToCompleteDefault
```

Comment:
Can these assertions all refer to the actual string that gets returned? For instance:
```
viewModel.sortHiddenTextLRN(messages) mustBe "Sort Local reference number (LRN) in descending order"
viewModel.sortHiddenTextDaysToComplete(messages) mustBe "Sort Days to complete in descending order"
```